src/components/Topics.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+import { getTopics } from '../../utils'
+import { Link } from 'react-router-dom'
+
+export default function Topics() {
+	const [topics, setTopics] = useState([])
+
+	useEffect(() => {
+		getTopics().then(({ topics }) => {
+			setTopics(topics)
+		})
+	}, [])
+
```

Comment:
Tried this on slow 3g. Would be helpful for the user to have a loading state for the topics as well 🙂